electricpy/__init__.py

```diff
@@ -4465,10 +4470,59 @@ def tap_changing_transformer(v1, v2, p, q, r, x):
         
     '''
 
-    ts = (v2 / v1) * (1 / (1 - (r * p + x * q) / (v1 * v2)))
+    ts = (Vgen*Vgen) / (Vgen*Vdis - (R * Pload + X * Qload) )
 
     return pow(ts, 0.5)
 
+def suspension_insulators(number_capacitors, capacitance_ratio, Voltage):
+    '''
+        This function calculates the voltage across each capacitor in a suspension insulator strain
```

Comment:
Consider specifying the heading of the docstring as: `Calculate the Voltage of Each Capacitor in a Suspension Insulator Strain`

Reply:
completed in 9e81883ef1b4b12f6e1d23e2badc681666b68951